hack/cherry_pick_pull.sh

```diff
@@ -162,7 +162,7 @@ gitamcleanup=true
 for pull in "${PULLS[@]}"; do
   echo "+++ Downloading patch to /tmp/${pull}.patch (in case you need to do this again)"
 
-  curl -o "/tmp/${pull}.patch" -sSL "https://github.com/${MAIN_REPO_ORG}/${MAIN_REPO_NAME}/pull/${pull}.patch"
+  curl -o "/tmp/${pull}.patch" -sSL "https://github.com/${MAIN_REPO_ORG}/${MAIN_REPO_NAME}/pull/${pull}.patch" -k
```

Comment:
Why do you think we need to change to the `insecure` way?

Reply:
curl client will verify the server certificate of github with ca certificate in the host,  so in my situation, the ca in my machine not ok; also this always hapy when in https proxy situation

Reply:
> curl client will verify the server certificate of github with ca certificate in the host, so in my situation, the ca in my machine not ok;

I don't think it solve this problem fundamentally.

Reply:
+1